sdk-extensions/aws/src/main/java/io/opentelemetry/sdk/extension/aws/trace/AwsXrayRemoteSampler.java

```diff
@@ -31,6 +33,18 @@
   private static final String WORKER_THREAD_NAME =
       AwsXrayRemoteSampler.class.getSimpleName() + "_WorkerThread";
 
+  // Unique per-process client ID, generated as a random string.
+  private static final String CLIENT_ID;
+
+  static {
```

Comment:
NIT - I'd put it as a private static method. Too much logic for a static init block, looks like an orphaned code ;)

Reply:
Sure